### PR TITLE
Convert wurstmeister images to bitnami images

### PR DESCRIPTION
**Pull request by ssingudasu**
# Description

Since wurstmeister is no longer maintained, there is a need to modify all wurstmeister zookeeper, kafka images to a well maintained docker hub

This PR is to 
1. convert all wurstmeister zookeeper/kafka images to bitnami
2. fix the workflows ci

## Local set up

`# docker-compose -f ./docker-compose.yml up -d zookeeper kafka1 kafka2 kafka3 kafka4 kafka5 kafka6`
`# make test`
`# make test-v2`

## make test result
Screenshot 2023-11-28 at 07 41 54

## make test-v2 result

Screenshot 2023-11-28 at 07 42 18

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -50,75 +50,83 @@ jobs:
...
       zookeeper:
-        image: wurstmeister/zookeeper
+        image: bitnami/zookeeper:latest
         ports:
           - "2181:2181"
+        env:
+          ALLOW_ANONYMOUS_LOGIN: yes
 
       kafka1:
-        image: wurstmeister/kafka:2.11-0.10.2.2
```
Comment (petedannemann) [inline .github/workflows/ci.yml:58]: This test block is titled `test010` and is supposed to test Kafka version 10.2

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -147,87 +155,95 @@ jobs:
...
       zookeeper:
-        image: wurstmeister/zookeeper
+        image: bitnami/zookeeper:latest
         ports:
           - "2181:2181"
+        env:
+          ALLOW_ANONYMOUS_LOGIN: yes
 
       kafka1:
-        image: wurstmeister/kafka:2.13-2.7.1
```
Comment (petedannemann) [inline .github/workflows/ci.yml:155]: This test block is titled test271 and is supposed to test Kafka version 2.7.1

File: docker-compose-auth.yml [hunk 3]
```diff
@@ -7,46 +7,57 @@
...
     ports:
       - "2181:2181"
+    environment:
+      ALLOW_ANONYMOUS_LOGIN: yes
 
   kafka:
-    image: wurstmeister/kafka:2.13-2.7.1
+    container_name: kafka
+    hostname: kafka
+    image: bitnami/kafka:2.7.0
```
Comment (petedannemann) [inline docker-compose-auth.yml:25]: Let's keep the Kafka version the same, we should use 2.7.1 here

File: docker-compose.yml [hunk 4]
```diff
@@ -1,116 +1,124 @@
-# By default, this docker-compose setup uses Kafka 2.4.1. This version can
-# be overwritten by setting the KAFKA_IMAGE_TAG environment variable; some choices here include:
+# By default, this docker-compose setup uses Kafka 2.5.0
```
Comment (petedannemann) [inline docker-compose.yml:1]: This comment is inaccurate.

File: docker-compose-auth.yml (hunk 3 above)
Comment (ssingudasu) [reply docker-compose-auth.yml:25]: bitnami kafka does not have 2.7.1. Resolving this

File: docker-compose.yml (hunk 4 above)
Comment (ssingudasu) [reply docker-compose.yml:1]: Will fix it

File: .github/workflows/ci.yml (hunk 2 above)
Comment (ssingudasu) [reply .github/workflows/ci.yml:155]: This has been fixed in the latest commit

File: docker-compose.yml (hunk 4 above)
Comment (ssingudasu) [reply docker-compose.yml]: This has been fixed in the latest commit

File: docker-compose.yml [hunk 5]
```diff
@@ -1,116 +1,124 @@
...
       - "2181:2181"
+    environment:
+      ALLOW_ANONYMOUS_LOGIN: yes
 
   # Zone 1 brokers
   kafka1:
-    image: wurstmeister/kafka:${KAFKA_IMAGE_TAG:-2.12-2.4.1}
+    container_name: kafka1
+    hostname: 169.254.123.123
+    image: bitnami/kafka:2.7.0
```
Comment (prithvi514) [inline docker-compose.yml:19]: Are we dropping support to other versions here by not using `${KAFKA_IMAGE_TAG}` ?

Comment (ssingudasu) [reply docker-compose.yml]: This has been included in recent commit

**Review by petedannemann — APPROVED** [review]

**Review by prithvi514 — APPROVED** [review]

**Pull request merged**